london_housing/Unit_4_Challenge_Tier_3.ipynb

```diff
@@ -0,0 +1,4406 @@
+{
```

Comment:
<p>I think you want .iloc[:,0] for first column here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Kazage/Data-Science-Notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>